webapp/certification/views.py

```diff
@@ -15,6 +15,36 @@
 )
 
 
+def certification_component_details(component_id):
+
+    try:
+        component = api.component_summary(component_id)
```

Comment:
If we can't recover from this exception, just abort with a 500, and report the exception to sentry.